user-interface/7003-MORD-manage_orders.md

```diff
@@ -1 +1,202 @@
+
+- if order type = `Limit`: **must** see the Limit [price](9001-DATA-data_display.md#quote-price) that was set on the order
+- if order type = `Market`: **must** not see a price for active or parked orders, a `-`, `Market` or `n/a` is more appropriate (API may return `0`).
+
+- **must** see the [time in force](9001-DATA-data_display.md#time-in-force) applied to the order (can be abbreviated here) (<a name="7003-MORD-006" href="#7003-MORD-006">7003-MORD-006</a>)
+- **should** see "created at" [time](9001-DATA-data_display.md#time)
+- **could** see updated at (this is used by the system when an order is amended, or repriced (in pegged and LP) not sure this in needed)
+
+- **should** see time priority (how many orders are before mine at this price)
+  
+- if the order is `Active` &amp; **not** part of a liquidity or peg shape: **must** see an option to [amend](#amend-order---price) the individual order (<a name="7003-MORD-007" href="#7003-MORD-007">7003-MORD-007</a>)
+- if the order is `Active` &amp; is part of a liquidity or peg shape: **must** **not** see an option to amend the individual order (<a name="7003-MORD-008" href="#7003-MORD-008">7003-MORD-008</a>)
+  - **could** see a link to amend shape
+- if the order is `Active` &amp; **not** part of a liquidity or peg shape: **must** see an option to [cancel](#cancel-orders) the individual order
+- if the order is `Active` &amp; is part of a liquidity or peg shape: **must** **not** see an option to cancel the individual order
+  - **could** see a link to cancel shape
+
+... so I can understand the state of my orders and decide what to do next
+### Filters
+
+When looking at a list of orders, I...
+
+- **should** have the ability to see all orders regardless of status in one list
+- **should** have the ability to see only active &amp; parked orders TODO update this based on what happens to parked orders
+- **should** have the ability to see only non-active &amp; parked order (i.e. all orders that do not have the status of Active &amp; Parked)
+- **could** have the ability to filter by any field(s)
+  - where a field is an enum: **should** be able to select one on or more values for a field that should be included
+
+... so I can focus on the type of orders statuses etc that I want to at that time
+
+### Sorting
+
+When looking at a list of orders, I...
+
+- **should** be able to sort the list (both directions) by any field in the order list
+  - **should** be able to add a secondary sort. e.g. by market then by date  
+- **should** have the default sorted by created time (or updated time if newer), with newest at the top
+- **should** retain sorting preferences between switching views / browser reload
+
+... so the sort order matches my mental model and or makes it easy for me to find the orders I am looking for  
+### Grouping
+
+When looking at a list of orders, I...
+
+- **should** be able to group orders by any field e.g. by market, lp etc
```

Comment:
What do you mean by grouping here. I see this as the same as sorting

Reply:
I mean grouping and sorting in the same way that excel does: https://edu.gcfglobal.org/en/excel2007/sorting-grouping-and-filtering-cells/1/

So I group all order by market. then sort the groups (maybe by expiry), with a secondary sort be price.

This way I can see all order by market (how I think about them), and then the ones with the in an order that makes sense